compiler/ir/serialization.common/src/org/jetbrains/kotlin/backend/common/serialization/IrFileDeserializer.kt

```diff
@@ -1009,7 +1009,8 @@ abstract class IrFileDeserializer(
                 if (proto.hasVarargElementType()) deserializeIrType(proto.varargElementType) else null,
                 flags.isCrossInline,
                 flags.isNoInline,
-                flags.isHidden
+                flags.isHidden,
+                false
```

Comment:
Shouldn't we put that flag into library too? I mean what if some plugin decides to use this?

Reply:
Yeah, we probably should. At this point it should never end up there, but I see no reason to not add support for it. Done.